src/components/organisms/ActiveTradeTable/ActiveTradeTable.tsx

```diff
@@ -214,64 +216,72 @@ export const ActiveTradeTable = ({
                             : OrderSide.LEND;
 
                     return (
-                        <div className='grid w-40 justify-center tablet:w-full'>
+                        <div className='flex h-5 w-40 items-center justify-center gap-1 tablet:w-fit'>
                             <div
-                                className={clsx('typography-caption w-full', {
-                                    'text-galacticOrange':
-                                        ccy && delistedCurrencySet.has(ccy),
-                                    'text-neutral7':
-                                        ccy && !delistedCurrencySet.has(ccy),
-                                })}
+                                className={clsx(
+                                    'typography-desktop-body-6 w-full',
+                                    {
+                                        'text-galacticOrange':
+                                            ccy && delistedCurrencySet.has(ccy),
+                                        'text-white':
+                                            ccy &&
+                                            !delistedCurrencySet.has(ccy),
+                                    }
+                                )}
                             >
                                 {getMaturityDisplayValue(
                                     maturityTimestamp,
                                     side,
                                     ccy
                                 )}
                             </div>
-                            <span className='typography-caption-2 h-5 w-full text-neutral-4'>
+                            <span className='w-full text-2.5 leading-3 text-neutral-400'>
                                 {formatDate(maturityTimestamp)}
                             </span>
                         </div>
                     );
                 },
                 header: tableHeaderDefinition(
-                    'Maturity',
-                    'Maturity of a loan contract is the date on which the contract is set to expire.'
+                    'Time to Maturity',
+                    'Maturity of a loan contract is the date on which the contract is set to expire.',
+                    'left'
                 ),
             }),
-            futureValueColumnDefinition(
+            priceYieldColumnDefinition(
                 columnHelper,
-                'FV',
-                'futureValue',
-                row => row.futureValue,
-                { color: true, priceList: priceList, compact: false },
-                'Future Value (FV) of a loan contract is the obligation value of the contract at time of maturity.'
+                'Mark Price',
```

Comment:
This should be "Market Price": https://www.figma.com/design/o2wjskNCdkt65haAgWPxo6/Product-UI-(Main)?node-id=16552-21060&m=dev

Reply:
https://www.figma.com/design/o2wjskNCdkt65haAgWPxo6/Product-UI-(Main)?node-id=16749-19251&m=dev
I am referring these designs